lib/Frontend/Frontend.cpp

```diff
@@ -512,8 +512,13 @@ bool CompilerInstance::setUpModuleLoaders() {
   // Wire up the Clang importer. If the user has specified an SDK, use it.
   // Otherwise, we just keep it around as our interface to Clang's ABI
   // knowledge.
+  SmallString<128> clangPath{ Invocation.getFrontendOptions().MainExecutablePath };
+  llvm::sys::path::remove_filename(clangPath);
+  llvm::sys::path::append(clangPath, "clang");
```

Comment:
Won't this do `build/{ninja|xcode}/swift-{target}/bin/clang` and not `build/{ninja|xcode}/llvm-{target}/bin/clang`? I think we want the latter.

To say this another way, the clang executable will be in the llvm build directory, not the swift build directory.

Reply:
Ah you're right, this doesn't help when building the contents of the Swift project itself. I'll think of a better way to do this.

Reply:
Well, it's not a regression at least, right? So we could potentially land this and then make a follow-up to fix that case.

Reply:
Not a regression indeed, but maybe I'll come up with a better solution quickly :) I'll try it this week & will let you know if I don't have another solution.

Reply:
OK. If you find a better solution I'll happily re-review. If not, I approved so you're welcome to land this.